### PR TITLE
Brought into compliance with semistandard 8.0.0 fixes #825

### LINKED ISSUE #825: Semistandard tests failing - PR proposal
When I run `npm test`, `semistandard` fails with a couple formatting errors. I made the changes and am ready to submit a PR but it just seemed so bizarre that I figured I'd ask about it here. To address semistandard's problems I:

1. Added `if(error)` to log errors to console in FeatureManager.js
2. Removed line padding around a single `if else` statement in Request.js

```
> semistandard --verbose src/**/*.js | snazzy

semistandard: Semicolons For All! (https://github.com/Flet/semistandard)

\Users\Tyler\WebstormProjects\esri-leaflet\src\Layers\FeatureLayer\FeatureManager.js
  68:27  error  Expected error to be handled  handle-callback-err

\Users\Tyler\WebstormProjects\esri-leaflet\src\Request.js
  125:4  error  Block must not be padded by blank lines  padded-blocks

✖ 2 problems

```

### PR DESCRIPTION
**Pull request by tyleralves**
See more detail in #825

### CODE REVIEW

**Comment by jgravois** [comment]
right now, our `package.json` ensures that the latest `7.x` release is what will be installed locally, so that will need to be bumped as well.
```json
"devDependencies": {
  "semistandard": "^7.0.5"
}
```
http://stackoverflow.com/questions/22343224/difference-between-tilde-and-caret-in-package-json

also, given your flurry of pull requests (and the cross contamination of unrelated commits), its probably time you start separating unique work in independent feature branches.

https://guides.github.com/introduction/flow/

**Comment by tyleralves** [comment]
Thanks John. Will resubmit clean PR with change to package.json.

File: src/Layers/FeatureLayer/FeatureManager.js [hunk 1]
```diff
@@ -70,6 +70,9 @@ export var FeatureManager = VirtualGrid.extend({
       if (supportedFormats && supportedFormats.indexOf('geoJSON') !== -1) {
         this.service.options.isModern = true;
       }
+      if (error) {
```
Comment (jgravois) [inline src/Layers/FeatureLayer/FeatureManager.js]: i can't repro the errors (even with semistandard `8.0.0`), but i think the change below would be sufficient and preferable to logging directly to the console.
```js
if (!error && supportedFormats && supportedFormats.indexOf('geoJSON') !== -1) {
```

**Comment by tyleralves** [comment]
Clean PR with @jgravois changes in #827

**Pull request closed without merging**